roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
```

Comment:
I think we have to figure out a way forward because I don't agree that it breaks consistency.